tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -356,6 +356,23 @@ public function testAssertCount($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideTestAssertCountCases
+     */
+    public function testAssertCountFromSame($expected, $input = null)
+    {
+        if (null === $input) {
+            $expected = str_replace('count', 'sizeOf', $expected);
```

Comment:
This will never run any test case with entirely lowercased `sizeof`, which is probably the most common usage. I'd rather rewrite the data provider so that different casings are tested just like with `count`.

Reply:
:+1: fixed by using some templating